ricoh-sp1xx-drv.py

```diff
@@ -248,15 +253,22 @@ def appendFile(fdst, fsrc, fbuffer_size=1024*16):
 #    Whitespace. 
 #    The height in pixels of the image, again in ASCII decimal. 
 #    Newline or other single whitespace character.
-def parsePbmSize(ffile):
-    lf = open(ffile,"rb")
-    lf.readline();   #log(">>>>"+lr) #magic P4
-    lf.readline();   #log(">>>>"+lrr) #comment
-    lrrr = lf.readline();  #log(">>>>"+lrrr) #width height
-    ls = lrrr.split(" ")
-    lf.close()
-    #return 4961,7016    
-    return int(ls[0]),int(ls[1])
+def parsePbmSize(ffile):     
+     proc = os.popen("pnmfile " + ffile + " | sed s/.*,//")
```

Comment:
Hi serkazak!
Why do you read pbm sizes in this way?
Are there any errors in mine?
pls explain this.
also - as i see from linux man pages

>   pnmfile - replaced by pamfile
>   This program is part of Netpbm(1).
>   pnmfile was replaced in Netpbm 10.9 (September 2002) by pamfile(1). 

so this program looks like something ancient.
if results are the same with mine style, my way is better, because doesn't use any additional program, and just reads
few first lines of PBM file, according to definition of this format. Is it wrong?